### PR TITLE
fix: switch to webpack for homepage building

### PR DESCRIPTION
**Pull request by radium-v**
# Description
Switch from yarn process spawning to webpack API to build the `fast-website` package directly in the `website` process.

## Motivation & context
The spawn process method doesn't work on Windows without changing some flags, and this method is safer.

## Issue type checklist
- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

The only caveat is that we lose the progress flag, but there may be ways to get stats back if we really miss it.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

I've tested it on Ubuntu (WSL2) and Windows (node and yarn directly installed) and both build and run successfully.

### CODE REVIEW

File: sites/website/src/build-homepage.js [hunk 1]
```diff
@@ -3,52 +3,30 @@
...
-yarnBuild.on("error", error => console.log(`error: ${error.message}`));
-
-yarnBuild.on("close", exitCode => {
-    console.log(`child process exited with code ${exitCode}`);
-    if (exitCode !== 0) {
-        process.exit(exitCode);
+console.log("Building fast-website with webpack...");
+webpack(fastWebsiteConfig, (err, stats) => {
+    if (err || stats.hasErrors()) {
+        console.log(err);
```
Comment (nicholasrice) [inline sites/website/src/build-homepage.js:20]: I'd kinda expect this to throw if there are errors or exit non-zero - is there a reason not to do that?

Comment (radium-v) [reply sites/website/src/build-homepage.js]: I updated the error checking so it will exit on stack errors and stat errors.

Comment (nicholasrice) [reply sites/website/src/build-homepage.js]: I think you want to exit `1`, sorry for the confusion: https://nodejs.org/api/process.html#process_process_exit_code

File: sites/website/src/build-homepage.js [hunk 2]
```diff
@@ -3,52 +3,43 @@
...
-    if (exitCode !== 0) {
-        process.exit(exitCode);
+    if (stats.hasWarnings()) {
+        info.warnings.forEach(w => console.warn(w));
     }
 
-    copyBundle();
+    console.info("Built fast-website to static");
+
+    copyfiles(
```
Comment (EisenbergEffect) [inline sites/website/src/build-homepage.js:40]: Is this call blocking? Does it matter? Just curious.

**Pull request merged**